From 70ae191f08b7495c5cd2d6c4ecb03467f3a900f7 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Fri, 3 Aug 2012 11:33:03 +0800
Subject: [PATCH 056/281] ENGR00221840-2 Disable sparse feature for ext4 image

JB uses sparse ext4 image by default, which is
used mostly on the emulator like virtualbox, qemu.
we don't support this feature, and require
the ext4 raw images be generated.

Change-Id: I0dc40ce72a8d4bd43de5f2018cea2896edbdacf0
Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 imx53_smd/BoardConfig.mk               | 2 ++
 imx6/arm2/BoardConfigArm2.mk           | 2 ++
 imx6/sabresd/SabreSDBoardConfigComm.mk | 2 ++
 3 files changed, 6 insertions(+)

diff --git a/imx53_smd/BoardConfig.mk b/imx53_smd/BoardConfig.mk
index d137515..ca9ea37 100755
--- a/imx53_smd/BoardConfig.mk
+++ b/imx53_smd/BoardConfig.mk
@@ -50,6 +50,8 @@ SENSOR_MMA8451 := true
 # for recovery service
 TARGET_SELECT_KEY := 28
 TARGET_USERIMAGES_USE_EXT4 := true
+# we don't support sparse image.
+TARGET_USERIMAGES_SPARSE_EXT_DISABLED := true
 
 # atheros 3k BT
 BOARD_USE_AR3K_BLUETOOTH := true
diff --git a/imx6/arm2/BoardConfigArm2.mk b/imx6/arm2/BoardConfigArm2.mk
index de52991..025527a 100644
--- a/imx6/arm2/BoardConfigArm2.mk
+++ b/imx6/arm2/BoardConfigArm2.mk
@@ -47,6 +47,8 @@ USE_QEMU_GPS_HARDWARE := false
 # for recovery service
 TARGET_SELECT_KEY := 28
 TARGET_USERIMAGES_USE_EXT4 := true
+# we don't support sparse image.
+TARGET_USERIMAGES_SPARSE_EXT_DISABLED := true
 
 # atheros 3k BT
 BOARD_USE_AR3K_BLUETOOTH := false
diff --git a/imx6/sabresd/SabreSDBoardConfigComm.mk b/imx6/sabresd/SabreSDBoardConfigComm.mk
index 03d8ce5..b092fa9 100755
--- a/imx6/sabresd/SabreSDBoardConfigComm.mk
+++ b/imx6/sabresd/SabreSDBoardConfigComm.mk
@@ -56,6 +56,8 @@ SENSOR_MMA8451 := true
 TARGET_SELECT_KEY := 28
 
 TARGET_USERIMAGES_USE_EXT4 := true
+# we don't support sparse image.
+TARGET_USERIMAGES_SPARSE_EXT_DISABLED := true
 
 # uncomment below lins if use NAND
 #TARGET_USERIMAGES_USE_UBIFS = true
-- 
1.8.0

